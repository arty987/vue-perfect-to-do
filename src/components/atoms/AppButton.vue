<script>
export default {
  name: "AppButton"
}
</script>

<template>
  <button type="button">
    <slot></slot>
  </button>
</template>