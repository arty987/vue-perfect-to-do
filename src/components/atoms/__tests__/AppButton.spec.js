import { shallowMount } from '@vue/test-utils'
import AppButton from '@/components/atoms/AppButton.vue';

describe('AppButton.vue', () => {
    it('Should render AppButton', () => {
        const wrapper = shallowMount(AppButton, {
            slots: {
                default: 'Button',
            }
        });
        expect(wrapper.element).toMatchSnapshot();
    });
})
