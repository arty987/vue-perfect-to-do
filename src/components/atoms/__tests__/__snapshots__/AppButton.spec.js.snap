// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppButton.vue Should render AppButton 1`] = `
<button
  class="app-button"
  type="button"
>
  
  Button
  
</button>
`;
